lib/info.js

```diff
@@ -144,7 +153,9 @@ function gotConfig(id, options, description, config, callback) {
     }
 
     info.formats = util.parseFormats(info);
-    info.description = description;
+
+    //add the additional to the infos and keep it future proof
+    info = Object.assign({}, info, additional);
```

Comment:
`Object.assign` is not in node v0.12, it's why the tests are failing for that version.

I'd argue that it might be time to stop supporting such an old version. But this functionality is also very simple to replicate with a for loop.